components/extractors/marketing-sales/customerio/index.md

```diff
@@ -73,72 +65,75 @@ Downloads campaigns dataset. Note that columns `actions` and `tags` contain JSON
 | timezone       |                                                          |
 
 ### Segments
+Segments are groups of people, subsets of your audience. You get information about segments and the customers 
+contained by a segment. 
 
-Segments are groups of people, subsets of your audience. You get get information about segments and the customers contained by a segment. 
-
-**Example output**
+Example output:
 
 | deduplicate_id | description                                                                                                                                                               | id | name             | progress | state    | tags |    type |
 |----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----|------------------|----------|----------|------|--------:|
 | 26521567:45:00 | Includes all people|  1 | Signed up        |          | finished |      | dynamic |
 | 26582936:15:00 | Anyone associated with active customer account.                                                                                                                           | 13 | Active Customers |          | finished |      | dynamic |
 
+Next, select the messages, customers, and/or activities that you wish to download.
 
 ### Messages
+This table contains metadata about messages. You may filter the messages by their type: `email`, `webhook`, `twilio`,
+`urban_airship`, `slack`, `push`.
 
-List metadata about messages. You may choose which types of messages you wish.  
-Allowable values are `email`, `webhook`, `twilio`, `urban_airship`, `slack`, `push`.
+Learn [more](https://customer.io/docs/api/#apibeta-apimessagesmessages_list). 
 
-[More info here](https://customer.io/docs/api/#apibeta-apimessagesmessages_list) 
+#### Incremental loading
+Some of the datasets allow the **Continue since last run** option. When checked, only those messages 
```

Comment:
když je to teda tady jen pod tím jedním datasetem, tak to pak nemá smysl tomu dávat nadpis a psát "some of the datasets"

Reply:
Jasne, upravim to. Dik